### PR TITLE
Adds a rudimentary system to summon equipment for distress teams, Tweaks the IAC team

### PR DESCRIPTION
**Pull request by fernerr**
- Makes it so you can add a map template to spawn equipment depending on which distress team that is summoned. Intended for the teams that spawn at the generic base.
 - Currently only IAC has this, the bottom spots in the distress prep base equipment area get some crates with rations, medical supplies tools and such, plus a light mech.
 - Tweaked the equipment loadout of IAC members to visually differentiate them a bit more. Replaced the bodyguard's suit with a heavy vest that matches better with the rest of their gear.
 - Gave the IAC members prefixes to better stand out from normal crewmembers.
 - Bumped doctor slots to 2.
IAC
Not pictured: gives the IAC a couple of medical voidsuits and a fireaxe intended for the paramedics.

### CODE REVIEW

File: code/modules/heavy_vehicle/premade/light.dm [hunk 1]
```diff
@@ -83,6 +83,20 @@
...
+	h_r_shoulder = /obj/item/mecha_equipment/mounted_system/flarelauncher
+	h_l_hand = /obj/item/mecha_equipment/mounted_system/medanalyzer
+	h_r_hand = /obj/item/mecha_equipment/clamp
+
+/mob/living/heavy_vehicle/premade/light/iac
+	name = "IAC exosuit"
+	desc = "A light and agile exosuit painted in the colours of the Interstellar Aid Corps."
+	icon_state = "odysseus"
+
+	e_color = "#96bcde"
```
Comment (mikomyazaki) [inline code/modules/heavy_vehicle/premade/light.dm]: Make a `COLOR_IAC` colour define next to `COLOR_TCFL`

File: maps/templates/distress/iac.dmm [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/spray/cleaner,
+/obj/item/deployable_kit/surgery_table,
+/obj/item/deployable_kit/surgery_table,
+/obj/item/storage/firstaid/surgery,
+/obj/item/storage/firstaid/surgery,
+/obj/machinery/iv_drip{
```
Comment (mikomyazaki) [inline maps/templates/distress/iac.dmm]: Is it intentional that these are different from normal iv_drips and you won't be able to push them?

Comment (fernerr) [reply maps/templates/distress/iac.dmm]: It's intentional, as the ship is small they'll be easier to load onboard. IVs are rarely pushed anyways and mostly pulled.

File: code/datums/ert/outsider.dm [hunk 3]
```diff
@@ -12,6 +12,7 @@
 	name = "Interstellar Aid Corps"
 	chance = 15
 	spawner = /datum/ghostspawner/human/ert/iac
+	equipment_map = new /datum/map_template/distress_iac
```
Comment (NonQueueingMatt) [inline code/datums/ert/outsider.dm]: Creating the equip map shouldn't be done here, really. Instead it should either be done when the teams are loaded or when the equipment map is.

File: code/modules/ghostroles/spawner/human/responseteams/iac.dm [hunk 4]
```diff
@@ -10,13 +11,15 @@
 /datum/ghostspawner/human/ert/iac/bodyguard
 	name = "IAC Bodyguard"
 	short_name = "iacbodyguard"
+	mob_name_prefix = "Bgd. "
```
Comment (NonQueueingMatt) [inline code/modules/ghostroles/spawner/human/responseteams/iac.dm]:
```suggestion
	mob_name_prefix = "Bdg. "
```

File: code/datums/ert/outsider.dm (hunk 3 above)
Comment (fernerr) [reply code/datums/ert/outsider.dm]: Handled.

**Comment by fernerr** [comment]
!review

**Pull request merged**